### PR TITLE
[javascript/en] Added setInterval

### PR DESCRIPTION
**Pull request by xssc**
Added the setInterval function provided by most browsers

### CODE REVIEW

File: javascript.html.markdown [hunk 1]
```diff
@@ -310,6 +310,12 @@ setTimeout(myFunction, 5000);
 // Note: setTimeout isn't part of the JS language, but is provided by browsers
 // and Node.js.
 
+// Another function provided by browsers is setInterval
+function myFunction(){
+    // this code will be called every 5 seconds
+}
+setInterval(myFunction(), 5000);
```
Comment (marcjansen) [inline javascript.html.markdown]: You meant `myFunction`, right? Without the braces?

**Comment by xssc** [comment]
yup, fixed, i'm a little to used to php (;

File: javascript.html.markdown [hunk 2]
```diff
@@ -310,6 +310,12 @@ setTimeout(myFunction, 5000);
 // Note: setTimeout isn't part of the JS language, but is provided by browsers
 // and Node.js.
 
+// Another function provided by browsers is setInterval
+function myFunction(){
```
Comment (vendethiel) [inline javascript.html.markdown]: please align the code style

**Comment by xssc** [comment]
I'm not sure what you mean, could you please elaborate? do you mean remove the white space and make it part of the 'note'?

**Comment by vendethiel** [comment]
I mean -- put the brace on the newline like the rest of the code in this file

**Comment by xssc** [comment]
I see only one place where the brace is on the next line, (line 249). The rest, the brace is on the same line

**Comment by vendethiel** [comment]
Sampling gone wrong! Sorry. Would you mind to fix this one instead, then, please?

**Comment by xssc** [comment]
yup, fixed

**Comment by vendethiel** [comment]
Great, thanks! :+1:

**Pull request merged**